addons/portal/views/portal_templates.xml

```diff
@@ -818,7 +818,7 @@
         <div t-if="isOffcanvas" class="offcanvas-header justify-content-end">
             <button type="button" class="btn-close text-reset" data-bs-dismiss="offcanvas" aria-label="Close"/>
         </div>
-        <div t-attf-class="{{'offcanvas-body' if isOffcanvas else 'mt-3'}}">
+        <div t-attf-class="{{'offcanvas-body' if isOffcanvas else 'mt-3 w-50'}}">
```

Comment:
Yes, indeed. With `mw-100`, the container takes its natural width and may expand a bit if there is a lot of text. Is that an issue?

I was considering forcing the container to always take all the available space but, when the address is short, the whole layout seems unaligned. I think that's the reason why they aligned the container to the right.

I think it's better to use all the available space when we need space :smiley: It should hence be okay to fill the large blank space between the two columns. The layout will then look fine when we have short and long addresses.

Reply:
Seems fine, but there's a point in the specifications mentioning something like, "The side cards with the contact details should always stay in the same place." Therefore, apply mw on the elements rather than the parent and let the parent occupy the available space but it will adjust according to the width of the elements if its small. Should we ask Ina if we should keep that behavior? Wdyt ?Thanks :)

Reply:
Ok, we can move the task forward and let Luc decide on the design.